src/domain/signupGroup/utils.ts

```diff
@@ -409,5 +409,7 @@ export const shouldCreatePayment = (
   ) > 0;
 
 export const canEditSignupGroup = (signupGroup: SignupGroup) =>
-  signupGroup.is_created_by_current_user ||
-  signupGroup.has_contact_person_access;
+  (signupGroup.is_created_by_current_user ||
```

Comment:
There probably can be a case where an individual signup in a group is being cancelled or refunded and a user then attempts to cancel the group. Should this also check if any signup in the group has a cancellation or a refund?

Reply:
Edit is now prevented also if signup group has at least 1 signup with payment cancellation/refund

Reply:
I also opened a new PR https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/417 to Linked Component UI repo to implement similar logic